### DISCUSSION TITLE
chrony -

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2247] [🇩🇪 Fragen&Antworten] [answered]
`Mar  5 14:41:42 Homematic daemon.err chronyd[1257]: Could not open /var/lib/chrony/chrony.drift.tmp : Permission denied`

jede Stunde taucht dieser Eintrag in den Logs auf, ist das ein Bug? 

**RaspberryMatic-3.67.10.20230303-dd916e-rpi3**

**Comment by Baxxy13** [comment, marked as answer]
Hmm, sehe ich auch.
Läuft chronyd vielleicht nicht mehr mit root-Rechten?

**Reply by jens-maus** [reply]
> Hmm, sehe ich auch.

Na dann los, ein Issue Ticket daraus machen wenn es da wirklich auch bei dir diese Probleme/Ausgaben gibt in den aktuellen nightly snapshots.
